libpod/container_internal.go

```diff
@@ -1361,19 +1361,57 @@ func (c *Container) addImageVolumes(ctx context.Context, g *generate.Generator)
 			continue
 		}
 		volumePath := filepath.Join(c.config.StaticDir, "volumes", k)
-		if _, err := os.Stat(volumePath); os.IsNotExist(err) {
+		srcPath := filepath.Join(mountPoint, k)
+
+		if _, err := os.Stat(srcPath); os.IsNotExist(err) {
+			logrus.Infof("Volume image mount point %s does not exits in root FS, need create it", k)
```

Comment:
"Volume image mount point %s does not exist in root FS, need to create it"

Reply:
Thanks.